app/assets/javascripts/discourse/app/components/user-nav.js

```diff
@@ -0,0 +1,18 @@
+import I18n from "I18n";
+
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+
+export default class UserNav extends Component {
+  @service currentUser;
+  @service site;
+  @service router;
+
+  get draftLabel() {
+    const count = this.currentUser.draft_count;
```

Comment:
Do we want this to be tracked/computed? Or does it not get dynamically updated anywhere?

Reply:
Ah good catch :+1: